site_scons/prereq_tools/base.py

```diff
@@ -894,7 +894,17 @@ def _setup_compiler(self, warning_level):
                         ['$CVS', '--add', '!**/src/vea/tests/'],
                         ['$CVS', '--add', '!**/src/vos/tests/'],
                         ['$CVS', '--add', '!**/src/engine/tests/'],
-                        ['$CVS', '--add', '!**/src/tests/']]
+                        ['$CVS', '--add', '!**/src/tests/'],
+                        ['$CVS', '--add', '!**/src/bio/smd/tests'],
+                        ['$CVS', '--add', '!**/src/cart/crt_self_test.h'],
+                        ['$CVS', '--add', '!**/src/cart/crt_self_test_client.c'],
+                        ['$CVS', '--add', '!**/src/cart/crt_self_test_service.c'],
+                        ['$CVS', '--add', '!**/src/client/api/tests'],
+                        ['$CVS', '--add', '!**/src/client/dfuse/test'],
+                        ['$CVS', '--add', '!**/src/gurt/examples'],
+                        ['$CVS', '--add', '!**/src/utils/crt_launch'],
+                        ['$CVS', '--add', '!**/src/utils/daos_autotest.c']
```

Comment:
(pylint-invalid-sequence-index) Sequence index is not an int, slice, or instance with __index__